app/views/static_pages/be_involved.html.erb

```diff
@@ -22,22 +22,21 @@
         <dd class="list--numbered__content">
           <%= raw t(
             'fincap.get_involved.list.use.content_html',
-            link_evaluation_toolkit: (link_to 'Evaluation Toolkit', 'evaluation-toolkit-homepage'),
+            link_evaluation_toolkit: (link_to 'Evaluation Toolkit', 'articles/evaluation-toolkit-overview'),
```

Comment:
Better way to link to an article page is to replace ```'articles/evaluation-toolkit-overview'``` with the rails route, like: ```article_path('en', 'evaluation-toolkit-overview')```